apps/manga-reader/web/src/components/menu.tsx

```diff
@@ -35,47 +35,68 @@ const MenuButton = ({
 export const Menu = (): JSX.Element => (
   <ul
     className={flex({
-      backgroundColor: "red",
+      backgroundColor: "blue.300",
       position: "sticky",
       bottom: "0",
       direction: "row",
       justify: "space-evenly",
       align: "center",
+      height: "100%",
     })}
   >
-    <li>
+    <li
+      className={css({
+        width: "18%",
+      })}
+    >
```

Comment:
**Suggestion:** Avoid hardcoding the width of each list item. Consider using a CSS class or a flexbox layout to evenly distribute the space.
```suggestion
    <li
      className={css({
        flex: 1,
      })}
    >
```